release/README.md

```diff
@@ -200,6 +200,16 @@ next 24 hours.
 (Optional) Create a PR to update the `google-cloud-cpp` package in
 [Conan](https://conan.io). These PRs are more involved.
 
+This package manager uses CMake, but ignores the library and target definitions
+exported by our CMake files. It reimplement the library definitions and
+dependencies in Python code. The recipes for each package does this for every
+package (gRPC, Protobuf, Boost, etc.) not just for our package recipe. For most
+packages, where there are only a handful of libraries, this is trivial, but our
+package has about 240 libraries (including the `*_protos` libraries).
+
+We use a helper script to automatically generate the Python data structures that
```

Comment:
Join these two paragraphs with "So, we ..."?

Reply:
Fixed.